PowerShell/ScubaGear/Modules/Support/Support.psm1

```diff
@@ -907,11 +907,11 @@ function New-Config {
 }
 
 Export-ModuleMember -Function @(
-    'Copy-ScubaBaselineDocument',
+    'Copy-SCuBABaselineDocument',
     'Install-OPA',
```

Comment:
I kinda think that we should update this function, too, only because it's generic and possible, if a bit unlikely, that some other program out there uses and installs OPA.  Maybe `Install-OPAforSCuBA` or something?